doc/intro.rst

```diff
@@ -293,6 +293,35 @@ To run a L2L simulation, copy the file :file:`bin/l2l-template.py` (see :doc:`l2
 initialization of the appropriate `Optimizers` and `Optimizees`. The rest of the code should be left in place for
 logging and recording. See the source of :file:`bin/l2l-template.py` for more details.
 
+Execution setup
+~~~~~~~~~~~~~~~
+The L2L framework works with JUBE in order to deploy the execution of the different instances of the optimizee on
+the available computational resources. This requires that the trajectory contains a parameter group called JUBE_params
+which contains details for the right execution of the program.
+
+In order to launch simulations on a laptop or a local cluster without a scheduler, only the mandatory parameters must
+be specified. These parameters are part of the template.
```

Comment:
From reading the text it is not clear for me what the mandatory parameters are.

Reply:
I will make this clearer and I also forgot to add the file with the scheduler parameters